From 7652626d749e3ae75d68f06fb88e705f26dc33fc Mon Sep 17 00:00:00 2001
From: Amy Chang <Amy_Chang@phoenix.com>
Date: Tue, 26 Apr 2022 09:54:23 +0000
Subject: [PATCH] Add dependency of phosphor-ipmi-net service

The network IPMI service should be run after xyz.openbmc_project.Network.service and multi-user.target,
to ensure the network is workable then start netipmid.
---
 phosphor-ipmi-net@.service | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/phosphor-ipmi-net@.service b/phosphor-ipmi-net@.service
index 77a800f..b3989fa 100644
--- a/phosphor-ipmi-net@.service
+++ b/phosphor-ipmi-net@.service
@@ -1,7 +1,10 @@
 [Unit]
 Description=Network IPMI daemon
 Requires=phosphor-ipmi-host.service
+Requires=xyz.openbmc_project.Network.service
 After=phosphor-ipmi-host.service
+After=xyz.openbmc_project.Network.service
+After=multi-user.target
 
 [Service]
 ExecStart=/usr/bin/netipmid -c %i
-- 
2.17.1

